### PR TITLE
Fix CR-151: bangla font changes

### PR DESCRIPTION
**Pull request by Nikhil77sutara**
# Changes
- prompt-text.ts

# How to test
- Check level 10 & 11 in bangla language

Ref: [CR-151](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-151)

[CR-151]: https://curiouslearning.atlassian.net/browse/CR-151?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/prompt-text.ts [hunk 1]
```diff
@@ -217,10 +207,32 @@ export class PromptText extends EventManager {
                 }
                 case "Word": {
                     if (this.levelData.levelMeta.protoType == "Visible") {
+                    if(this.targetStones.length!=this.currentPromptText.length){
+                    if(this.targetStones.length>i){   
+                    if (this.droppedStoneCount > i || this.droppedStoneCount== undefined) {
+                        this.context.fillStyle = "black";
```
Comment (ashwinnair-chimple) [inline src/components/prompt-text.ts:213]: Code can be refactored and handled easily

this.context.fillStyle = (this.droppedStoneCount > i || this.droppedStoneCount === undefined) ? "black" : "red";

File: src/components/prompt-text.ts [hunk 2]
```diff
@@ -217,10 +207,32 @@ export class PromptText extends EventManager {
...
+                        this.context.fillText(
```
Comment (ashwinnair-chimple) [inline src/components/prompt-text.ts:214]: Instead of using the same logic twice in if else,use it once under the above code.

Comment (ashwinnair-chimple) [inline src/components/prompt-text.ts:214]: This logic can be just used once in this if statement , if the refactored with the above code.

File: src/components/prompt-text.ts (hunk 1 above)
Comment (Nikhil77sutara) [reply src/components/prompt-text.ts]: Thanks @ashwinnair-chimple  for you feedback. I have made all the necessary  changes requested by you🫡.

File: src/components/prompt-text.ts [hunk 3]
```diff
@@ -217,23 +207,26 @@ export class PromptText extends EventManager {
...
-                            startPrompttextX,
+                            this.targetStones[i],
+                            startPrompttextX+startPrompttextX/10,
                             y
-                        );
-                    } else {
-                        this.context.fillStyle = "red";
+                        );    
+                }
+                break;
```
Comment (ashwinnair-chimple) [inline src/components/prompt-text.ts:219]: you can remove this break also @Nikhil77sutara

**Pull request merged**